server/src/main/java/org/elasticsearch/index/mapper/ObjectMapper.java

```diff
@@ -87,6 +86,64 @@ public Builder add(Mapper.Builder builder) {
             return this;
         }
 
+        public Builder addMappers(Map<String, Mapper> mappers) {
```

Comment:
Looks like this can be package private, seems only be used by ObjectMapperMergeTests